docs/v1/P4Runtime-Spec.mdk

```diff
@@ -878,6 +900,48 @@ message PkgInfo {
   string url = 8;
 }
 ~ End Proto
+
+### Annotating P4 code with PkgInfo
+A P4 progam's `PkgInfo` may be declared using one more more of the following
+annotations, attached to the `main` block only:
+
+~ Begin p4example
+@pkginfo(key=value)
+@pkginfo(key=value[,key=value,...])
+~ End p4example
+
+Multiple `@pkginfo` annotations are allowed. For compactness, multiple key-value
+pairs can appear in a single `@pkginfo` annotation, separated by commas. Each
+key should only appear once, but if it appears multiple times, then the last
+value shall be used. Declaring one or more of these annotations will generate a
+single corresponding `PkgInfo` message in the P4Info as described in
+[PkgInfoMessage](#sec-pkginfo-message). The `key`s must be from among the
+message elements inside `PkgInfo`, for example, `name`, `version`, etc. One
+exception is that the `Documentation` element of `PkgInfo` must be expressed as
```

Comment:
shouldn't we use `@brief` and `@description` for documentation annotations? It may be more consistent.

Reply:
In other words, if you add those annotations to main() they populate the PkgInfo? Sounds good. In fact I was vacillating over this when I wrote it.